### PR TITLE
using case insensitive keys for fqdn maps and ports list AND adding v…

### PR DESCRIPTION
**Pull request by anaggar**
…erbose logs for them

### CODE REVIEW

**Comment by anaggar** [comment]
already have a test will provide rg name in caps in existing test

File: Tasks/AzureFileCopy/Utility.ps1 [hunk 1]
```diff
@@ -431,15 +431,23 @@ function Get-MachineNameFromId
 
     if($map)
     {
+        Write-Verbose "Map for $mapParameter : " -Verbose
```
Comment (kmkumaran) [inline Tasks/AzureFileCopy/Utility.ps1]: Check if you can wrap the write-verbose to come up only when debug flag is set.

**Comment by anaggar** [comment]
@pavanadepu2: we have always taken a dependency on a platform to get Data, in this case data is already with us. now point is that data is not exact (user initially provided while creating and the one he gets from dropdown), Now given that resource group names in azure portal are case insensitive, then we should not add any other platform call as it might even break in future version(we are not sure). So in terms of  performance and less dependency on platform we shud avoid azure call because we already have data with us.

File: Tasks/AzureFileCopy/Utility.ps1 [hunk 2]
```diff
@@ -485,30 +493,36 @@ function Get-MachinesFqdnsForPublicIP
         {
             if(-not [string]::IsNullOrEmpty($publicIP.DnsSettings.Fqdn))
             {
-                $fqdnMap[$publicIp.IpConfiguration.Id] =  $publicIP.DnsSettings.Fqdn
+                $fqdnMap[$publicIp.IpConfiguration.Id.ToLower()] =  $publicIP.DnsSettings.Fqdn
```
Comment (kmkumaran) [inline Tasks/AzureFileCopy/Utility.ps1]: While I understand the need for tolower() for VMid, I am not sure about rest of resources. Ensure to test that we are not breaking for rest of the resources.

Comment (anaggar) [inline Tasks/AzureFileCopy/Utility.ps1]: Tested working fines for NICs, LBs, publicIPs also

**Comment by kmkumaran** [comment]
Looks fine.

**Pull request merged**